server/const.go

```diff
@@ -1,5 +1,9 @@
 package main
 
 const (
-	PluginName = "mattermost-plugin-metrics"
+	PluginName    = "mattermost-plugin-metrics"
+	tsdbDirName   = "data"
```

Comment:
`data` seems a bit too generic. Maybe `tsdb`? 0/5

Reply:
'data' is the default directory used for Prometheus as well, since it will always be under the `mattermost-plugin-metrics` I just wanted to keep the convention similar.